### PR TITLE
Create a service to Vote

### LINKED ISSUE #5: Create a service to Vote
As a user, I want to vote on the story. So I need to get the votes information of the entire room and know the result.

- Test cases are required to finish this.
- Remember you just need to do the necessary.

### PR DESCRIPTION
**Pull request by felipesena**
I want to resolve #5

### CODE REVIEW

**Comment by felipesena** [comment]
@gscavalcante please take a look at the Javadoc that i created in the interface

File: src/main/java/com/gcavalcante/uaiplanningpoker/service/VoteService.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+     * @param vote
+     * @return Vote
+     */
+    Vote createVote(Vote vote);
+
+    /**
+     * Retrieve all votes by story id
+     *
+     * @param storyId
+     * @return Set<Vote>
```
Comment (gscavalcante) [inline src/main/java/com/gcavalcante/uaiplanningpoker/service/VoteService.java]: You can put here: `@return all votes by story id`

File: src/main/java/com/gcavalcante/uaiplanningpoker/service/VoteService.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+     *
+     * @param vote
+     * @return Vote
+     */
+    Vote createVote(Vote vote);
+
+    /**
+     * Retrieve all votes by story id
+     *
+     * @param storyId
```
Comment (gscavalcante) [inline src/main/java/com/gcavalcante/uaiplanningpoker/service/VoteService.java]: The param description is empty

File: src/main/java/com/gcavalcante/uaiplanningpoker/service/VoteService.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+import java.util.Set;
+
+public interface VoteService {
+
+    /**
+     * Create a new vote.
+     *
+     * @param vote
+     * @return Vote
```
Comment (gscavalcante) [inline src/main/java/com/gcavalcante/uaiplanningpoker/service/VoteService.java]: The return is empty. Maybe: `@return a persisted vote`.

File: src/main/java/com/gcavalcante/uaiplanningpoker/service/VoteService.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+import com.gcavalcante.uaiplanningpoker.errors.ResourceNotFoundException;
+
+import java.util.Set;
+
+public interface VoteService {
+
+    /**
+     * Create a new vote.
+     *
+     * @param vote
```
Comment (gscavalcante) [inline src/main/java/com/gcavalcante/uaiplanningpoker/service/VoteService.java]: The description is empty.

File: src/main/java/com/gcavalcante/uaiplanningpoker/repository/VoteRepository.java [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.Set;
+
+public interface VoteRepository extends CrudRepository<Vote, Long> {
+
+    /**
+     * Get all votes by Story id
+     *
+     * @param storyId
```
Comment (gscavalcante) [inline src/main/java/com/gcavalcante/uaiplanningpoker/repository/VoteRepository.java]: The description is empty. Normally, I read the Spring repository class to understand how they add a JavaDoc to these parameters.

**Pull request merged**